definitions/UserDefinitions.php

```diff
@@ -75,8 +80,31 @@ public static function getGroup(Group $group)
             'description' => $group->description,
             'show_at_registration' => $group->show_at_registration,
             'show_at_directory' => $group->show_at_directory,
-            'sort_order' => $group->sort_order
+            'sort_order' => $group->sort_order,
         ];
     }
-}
 
+    public static function getAllUserData(User $user): array
```

Comment:
@yurabakhtin See proposal regarding Module API. Then we could use the identical callback in the Rest Module as in the Wiki module, for example.

Reply:
It is closed because of new solution https://github.com/humhub-contrib/legal/pull/77#issuecomment-2285994673